__tests__/topics.test.js

```diff
@@ -0,0 +1,45 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const testData = require('../db/data/test-data/index');
+const seed = require('../db/seeds/seed');
+const sorted = require('jest-sorted');
+
+beforeEach(() => {
+  return seed(testData)
+})
+
+afterAll(() => {
+    db.end();
+  });
+
+  describe('1. GET /api/topics', () => {
+    test('status:200, responds with an array of topic objects', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+          const { topics } = body;
+          expect(topics).toBeInstanceOf(Array);
+          
+          topics.forEach((topic) => {
+            expect(topic).toEqual(
+              expect.objectContaining({
+                description: expect.any(String),
+                slug: expect.any(String)
+              })
+            );
```

Comment:
Could you please add a test for the length of the array. If you test an empty array with a forEach, because it has no elements it will never technically fail so this could pass even if topics is an empty array which is not what we want